src/extension/js/add-polyratings.js

```diff
@@ -205,6 +189,34 @@ export default class PolyratingIntegrator {
     })
   }
 
+  checkCalPolyRatings(rawname, nameElems) {
+    const [lastName, firstName] = rawname.split(/[, ]/)
+      .filter(name => name.length > 2 && name.split('I').length !== name.length + 1) // detects II and M.
+
+    const proxyUrl = 'https://cors-anywhere.herokuapp.com/'
+    const herf = `https://www.calpolyratings.com/${firstName.toLowerCase()}-${lastName.toLowerCase()}/`
+
+    fetch(proxyUrl + herf).then(res => res.text()).then(data => {
```

Comment:
Please use the chrome extension api for making http calls, as is done earlier in this file.

```
chrome.runtime.sendMessage({
  method: 'GET',
  action: 'xhttp',
  url: 'YOUR_URL',
  data: '',
},
response => {
  // do stuff here
})
```